okhttp/src/test/java/okhttp3/WholeOperationTimeoutTest.java

```diff
@@ -286,7 +287,10 @@ public void timeoutFollowingRedirectOnNewConnection() throws Exception {
     }
   }
 
+  @Flaky
```

Comment:
To be clear it shouldn't be flaky after this point, but this will record the events for every test run, so we can observe typical response times.

Reply:
👍🏻